### PR TITLE
updates to authoring.md

### PR DESCRIPTION
**Pull request by angelazhangmsft**
add link to sample and some other edits

### CODE REVIEW

File: docs/authoring.md [hunk 1]
```diff
@@ -1,49 +1,51 @@
...
 --- |
 `net5.0-windows10.0.19041.0` |
 `net5.0-windows10.0.18362.0` |
 `net5.0-windows10.0.17763.0` |
 
 The library you are authoring should specify the following properties in its project file: 
+
 ``` csproj
 <PropertyGroup>
   <!-- update the Windows API version to reflect your TargetFramework -->
```
Comment (j0shuams) [inline docs/authoring.md:30]: "Choose your TargetFramework for the desired Windows SDK projection"

or just remove the comment if you think that's better

File: docs/authoring.md [hunk 2]
```diff
@@ -1,49 +1,51 @@
...
+<PropertyGroup>
+  <GeneratePackageOnBuild>true</GeneratePackageOnBuild> 
+</PropertyGroup>
+```
 
-**If you are going to write your own nuspec, i.e. not rely on automatic packaging** then the CsWinRT target that adds the hosting dlls to your package will not run, and you should make sure your nuspec contains the following ```file``` entries for ```MyAuthoredComponent``` (note: your TargetFramework may vary). If adding a file entry for ```Coords.targets``` does not work, then update the project file per the above example. 
+**If you are going to write your own nuspec, i.e. not rely on automatic packaging** then the CsWinRT target that adds the hosting dlls to your package will not run, and you should make sure your nuspec contains the following ```file``` entries for ```MyAuthoredComponent``` (note: your TargetFramework may vary). 
 
 ``` nuspec
 <files>
```
Comment (j0shuams) [inline docs/authoring.md:51]: Let's add a comment around line 60 (gh wouldnt let me add this comment there)

that says "`buildTransitive` is for consumers using packagereference, `build` is for consumers using packages.config"

**Review by j0shuams — APPROVED** [review]
Sweet!

File: docs/authoring.md [hunk 3]
```diff
@@ -1,49 +1,51 @@
...
+## Generate a NuGet package for the component
+You can add the following property to the library project file to automatically generate a NuGet package.
 
-## Using your authored component
-To use the component in a C# app, the authored component just needs to be added as a project/package reference.
-
-For native (C++) apps, there are DLLs needed to host your authored component. When you use the automatic nuget packaging on-build support (in Visual Studio) to make a nupkg for your runtime component, the DLLs/WinMD are automatically added to your nupkg, before the ```GenerateNuspec``` MSBuild step.
+``` csproj
+<PropertyGroup>
+  <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
```
Comment (j0shuams) [inline docs/authoring.md:44]: Also, this is kind of misleading. You don't have to specify this for our targets to go into work. You could just right click the project and click "Pack" and our packaging logic kicks in.

**Review by manodasanW — APPROVED** [review]

**Pull request merged**